### PR TITLE
core/logger/audit: headers toml format

### PR DESCRIPTION
**Pull request by jmank88**
https://app.shortcut.com/chainlinklabs/story/55477/toml-config-auditlogger-headers-format

### CODE REVIEW

**Comment by obelisk** [comment]
So the idea would be that the configuration files have a slightly different flow than ENV variables where configuration files already have the headers isolated but need splitting, where the ENV variables still split on the backslash character?

**Comment by jmank88** [comment]
> So the idea would be that the configuration files have a slightly different flow than ENV variables where configuration files already have the headers isolated but need splitting, where the ENV variables still split on the backslash character?

Yes, but FWIW there is no need to worry about alignment between these two or even to bikeshed on the ENV format at all because it will be removed after new config is rolled out. (and the conversion happens automatically via dump)

**Comment by obelisk** [comment]
Sounds good to me

File: core/config/v2/docs/core.toml [hunk 1]
```diff
@@ -113,7 +113,7 @@ ForwardToUrl = 'http://localhost:9898' # Example
 # JsonWrapperKey if set wraps the map of data under another single key to make parsing easier
 JsonWrapperKey = 'event' # Example
 # Headers is the set of headers you wish to pass along with each request
-Headers = 'Authorization||token\X-SomeOther-Header||value with spaces | and a bar+*' # Example
+Headers = ['Authorization: token', 'X-SomeOther-Header: value with spaces | and a bar+*'] # Example
```
Comment (samsondav) [inline core/config/v2/docs/core.toml:116]: That is quite an improvement

**Review by samsondav — APPROVED** [review]

**Pull request merged**